ceph.spec.in

```diff
@@ -1076,6 +1085,10 @@ This package provides Ceph’s default alerts for Prometheus.
 # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=48200
 %define _lto_cflags %{nil}
 
+%if 0%{?with seastar} == 8
```

Comment:
(just for my information...)  What is the '8' here? where is it set?

Reply:
fixed and repushed.